src/applications/disability-benefits/all-claims/utils/validations.js

```diff
@@ -0,0 +1,110 @@
+import { convertToDateField } from '~/platform/forms-system/src/js/validation';
+import { isValidDateRange } from '~/platform/forms/validations';
+
+const messages = {
+  invalidRange: 'Enter a service start date that’s before your end date.',
+  endDate1990: 'Enter a service end date after August 2, 1990.',
+  endDate2001: 'Enter a service end date after September 11, 2001.',
+  // This is largely redundant because the component validation already prevents
+  // inputting dates in the future. However, this is in place just in case, because...
+  // TODO: The current schema validation says that the Veteran can input a year
+  // up to 100 years in the future. That is incorrect and should be corrected/removed.
+  futureDate: 'Enter a service year that’s before or during the current year.',
+  // Schema validations will display an error when a value is missing, however it
+  // will not properly prevent the Veteran from progressing, necessitating this.
+  // TODO: The error message here can be made more specific, but that will somewhat
+  // depend on what decisions (and schema updates) are made in regards to partial dates.
+  missingValue: 'Enter a service date that includes the month, day, and year.',
```

Comment:
Each of these messages require content changes to match what is presented in https://www.figma.com/design/uffC3A5H3rwEtjSybqG84G/526ez---Source-of-truth-updated-5.2025?node-id=1282-33795&p=f&t=m94nnHpZjZ1BKHOn-0. In addition, it appears that there will need to be a specific `futureDate` message for each start and end date entries.